apps/mull-api/src/app/auth/auth.service.ts

```diff
@@ -49,6 +49,7 @@ export class AuthService {
     res.cookie('mullToken', refreshToken, {
       maxAge: 7 * 24 * 3600 * 1000, // 7 days in ms
       secure: environment.production, // Determines if cookie is to be used with HTTPS only
+      httpOnly: true,
```

Comment:
Just a question, won't these two interfere with each other, judging from the comment that was left?

Reply:
No, there's a bit of a misconception with the naming of the option. `httpOnly` refers to how and where the cookie is used. This is mainly to allow the cookie to be used only when communicating with the HTTP/S protocols. Meaning it cannot be accessed in javascript. https://owasp.org/www-community/HttpOnly